vitamin/src/main/res/values/styles.xml

```diff
@@ -298,13 +356,392 @@
+        <item name="vtmnTagAlertInteractiveStyle">@style/Widget.Vitamin.Tag.Alert.Interactive</item>
+        <item name="vtmnTagBrandStyle">@style/Widget.Vitamin.Tag.Brand</item>
+        <item name="vtmnTagBrandInteractiveStyle">@style/Widget.Vitamin.Tag.Brand.Interactive</item>
+        <item name="vtmnTagDecorativeGravelStyle">@style/Widget.Vitamin.Tag.DecorativeGravel</item>
+        <item name="vtmnTagDecorativeGravelInteractiveStyle">
+            @style/Widget.Vitamin.Tag.DecorativeGravel.Interactive
+        </item>
+        <item name="vtmnTagDecorativeBrickStyle">@style/Widget.Vitamin.Tag.DecorativeBrick</item>
+        <item name="vtmnTagDecorativeBrickInteractiveStyle">
+            @style/Widget.Vitamin.Tag.DecorativeBrick.Interactive
+        </item>
+        <item name="vtmnTagDecorativeSaffronStyle">@style/Widget.Vitamin.Tag.DecorativeSaffron</item>
+        <item name="vtmnTagDecorativeSaffronInteractiveStyle">
+            @style/Widget.Vitamin.Tag.DecorativeSaffron.Interactive
+        </item>
+        <item name="vtmnTagDecorativeGoldStyle">@style/Widget.Vitamin.Tag.DecorativeGold</item>
+        <item name="vtmnTagDecorativeGoldInteractiveStyle">@style/Widget.Vitamin.Tag.DecorativeGold.Interactive
+        </item>
+        <item name="vtmnTagDecorativeJadeStyle">@style/Widget.Vitamin.Tag.DecorativeJade</item>
+        <item name="vtmnTagDecorativeJadeInteractiveStyle">@style/Widget.Vitamin.Tag.DecorativeJade.Interactive
+        </item>
+        <item name="vtmnTagDecorativeEmeraldStyle">@style/Widget.Vitamin.Tag.DecorativeEmerald</item>
+        <item name="vtmnTagDecorativeEmeraldInteractiveStyle">
+            @style/Widget.Vitamin.Tag.DecorativeEmerald.Interactive
+        </item>
+        <item name="vtmnTagDecorativeCobaltStyle">@style/Widget.Vitamin.Tag.DecorativeCobalt</item>
+        <item name="vtmnTagDecorativeCobaltInteractiveStyle">
+            @style/Widget.Vitamin.Tag.DecorativeCobalt.Interactive
+        </item>
+        <item name="vtmnTagDecorativeAmethystStyle">@style/Widget.Vitamin.Tag.DecorativeAmethyst</item>
+        <item name="vtmnTagDecorativeAmethystInteractiveStyle">
+            @style/Widget.Vitamin.Tag.DecorativeAmethyst.Interactive
+        </item>
+        <!-- endregion -->
+
+        <!-- region Tabs -->
+        <!-- region Vitamin Tabs -->
+        <item name="vtmnTabStyle">@style/Widget.Vitamin.TabLayout</item>
+        <item name="vtmnTabLeadingIconStyle">@style/Widget.Vitamin.TabLayout.LeadingIcon</item>
+        <item name="vtmnTabTopIconStyle">@style/Widget.Vitamin.TabLayout.TopIcon</item>
+        <!-- endregion -->
+        <!-- region Material Tabs -->
         <item name="tabStyle">?attr/vtmnTabStyle</item>
+        <!-- endregion -->
+        <!-- endregion -->
     </style>
+
+    <style name="Theme.Vitamin.DayNight" parent="Theme.Vitamin.Light"/>
+
+    <!-- {@deprecated Use Theme.Vitamin.x instead.} -->
```

Comment:
You can specify `Theme.Vitamin.DayNight` here because that was the equivalent before this contribution.